src/main/kotlin/uk/gov/justice/digital/assessments/jpa/entities/assessments/AssessmentEntity.kt

```diff
@@ -16,21 +17,21 @@ import javax.persistence.OneToMany
 import javax.persistence.Table
 
 @Entity
-@Table(name = "ASSESSMENT")
+@Table(name = "assessment", schema = "hmppsassessmentsapi")
 class AssessmentEntity(
```

Comment:
aligning entities to the actual casing we have in live, some of our entities were using lowercase and some uppercase so i decided to go for lowercase which reflects how it is in the actual db